wazo_confd/plugins/schedule/schema.py

```diff
@@ -116,15 +116,15 @@ class ScheduleSchema(BaseSchema):
     enabled = fields.Boolean()
     links = ListLink(Link('schedules'))
 
-    incalls = Nested('IncallSchema', only=['id', 'links'], many=True, dump_only=True)
+    incalls = Nested('IncallSchema', only=['id', 'description', 'extensions', 'links'], many=True, dump_only=True)
```

Comment:
I guess the `description` (`extensions` too?) field is here because the `incall` object doesn't have `label`/`name`, so you use `description` field as name.
The problem here is if we merge as it, those fields will become part of the specification and we won't be able to remove them before very long time ...
The proper solution would be to add a new `label` field